### PR TITLE
dev-util/sh: add manpage, add live build, and add co-maintianer

### PR DESCRIPTION
**Pull request by ran-dall**
This commits fixes the generation and deployment of the `sh` man page, as well as add a live build for `sh`.

It also adds me (@ran-dall) as a co-maintainer for this package.

### CODE REVIEW

File: dev-util/sh/sh-3.5.1.ebuild [hunk 1]
```diff
@@ -7,17 +7,40 @@ inherit go-module
 
 DESCRIPTION="A shell parser, formatter, and interpreter with bash support"
 HOMEPAGE="https://github.com/mvdan/sh"
-SRC_URI="https://github.com/mvdan/sh/archive/v${PV}.tar.gz -> ${P}.tar.gz"
-SRC_URI+=" https://dev.gentoo.org/~sam/distfiles/${CATEGORY}/${PN}/${P}-deps.tar.xz"
+
+if [[ ${PV} == *9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/mvdan/sh.git"
+	RESTRICT="fetch mirror test"
```
Comment (thesamesam) [inline dev-util/sh/sh-3.5.1.ebuild:14]: Please drop the RESTRICT. Tests work fine IIRC.

Also, please swap the order of EGIT_REPO_URI and inherit (sam style).

File: dev-util/sh/sh-3.5.1.ebuild [hunk 2]
```diff
@@ -27,4 +50,8 @@ src_test() {
 
 src_install() {
 	dobin shfmt
+	if use man; then
+		doman shfmt.1
+	fi
+
```
Comment (thesamesam) [inline dev-util/sh/sh-3.5.1.ebuild:56]:
```suggestion
```

File: dev-util/sh/metadata.xml [hunk 3]
```diff
@@ -5,6 +5,10 @@
 		<email>sam@gentoo.org</email>
 		<name>Sam James</name>
 	</maintainer>
+	<maintainer type="person">
```
Comment (thesamesam) [inline dev-util/sh/metadata.xml:8]:
```suggestion
	<maintainer type="person" proxied="yes">
```

File: dev-util/rustup/metadata.xml [hunk 4]
```diff
@@ -9,6 +9,9 @@
 		<email>rust@gentoo.org</email>
 		<name>Rust Project</name>
 	</maintainer>
+	<longdescription lang="en">
```
Comment (thesamesam) [inline dev-util/rustup/metadata.xml:12]: wrong package? :)

Comment (ran-dall) [reply dev-util/rustup/metadata.xml:12]: Oh, WTF? LOL. My bad.

File: dev-util/sh/sh-3.5.1.ebuild [hunk 5]
```diff
@@ -7,17 +7,40 @@ inherit go-module
 
 DESCRIPTION="A shell parser, formatter, and interpreter with bash support"
 HOMEPAGE="https://github.com/mvdan/sh"
-SRC_URI="https://github.com/mvdan/sh/archive/v${PV}.tar.gz -> ${P}.tar.gz"
-SRC_URI+=" https://dev.gentoo.org/~sam/distfiles/${CATEGORY}/${PN}/${P}-deps.tar.xz"
+
+if [[ ${PV} == *9999 ]]; then
+	EGIT_REPO_URI="https://github.com/mvdan/sh.git"
+	inherit git-r3
+	RESTRICT="fetch mirror"
```
Comment (thesamesam) [inline dev-util/sh/sh-3.5.1.ebuild:14]: drop

File: dev-util/sh/sh-3.5.1.ebuild [hunk 6]
```diff
@@ -7,17 +7,40 @@ inherit go-module
...
+else
+	SRC_URI="https://github.com/mvdan/sh/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+	SRC_URI+=" https://dev.gentoo.org/~sam/distfiles/${CATEGORY}/${PN}/${P}-deps.tar.xz"
+	KEYWORDS="~amd64 ~arm64 ~x86"
+	RESTRICT="mirror"
```
Comment (thesamesam) [inline dev-util/sh/sh-3.5.1.ebuild:19]: drop

**Pull request closed without merging**